### DISCUSSION TITLE
Proxychains with nuclei

### DISCUSSION THREAD

**Discussion by Shafique-wasta** [projectdiscovery/nuclei#3939] [Q&A]
how we can use proxychains with nuclei scan.

**Comment by MetzinAround** [comment]
Hi @Shafique-wasta , Could you tell us a bit more about what proxychains is and what your use case might be?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
